Source/utils/GRParmParse.hpp

```diff
@@ -127,6 +129,65 @@ class GRParmParse : public ParmParse
              std::vector<data_t>(num_comp, default_value));
     }
 
+    static bool folder_exists(const std::string &path)
```

Comment:
Do you think that this does not make it ambiguous in terms of thinking in the future that this should also return true for files? Opposed to just for directories. As it is, for the path to a given *file* (even if it exists) this will return false. If you think this won't cause confusion then I'm fine.

Reply:
I see your point. What about `directory_path_exists`?